packages/manager/apps/pci-private-network/src/pages/new/steps/ConfigurationStep.tsx

```diff
@@ -390,7 +393,21 @@ export default function ConfigurationStep({
                     level={ODS_TEXT_LEVEL.body}
                     size={ODS_TEXT_SIZE._400}
                   >
-                    {t('pci_projects_project_network_private_create_vlan_tip')}
+                    <Trans
+                      t={t}
+                      i18nKey="pci_projects_project_network_private_create_vlan_tip"
+                      components={{
+                        link: (
+                          <a
+                            href={VLAN_GUIDE_URL}
+                            target="_blank"
+                            rel="noreferrer"
+                          >
+                            {tCommon('common_find_out_more_here')}
+                          </a>
+                        ),
+                      }}
+                    />
```

Comment:
I don't want block this PR but why don't use a simple : 

```suggestion
                    {t('pci_projects_project_network_private_create_vlan_tip")}
                    <a
                      href={VLAN_GUIDE_URL}
                      target="_blank"
                      rel="noreferrer"
                    >
                      {tCommon('common_find_out_more_here')}
                    </a>
```

?

Reply:
And replace `<a />` by Link component for have unified behaviour 🙂  
`packages/manager-react-components/src/components/typography/links/links.component.tsx`

https://ovh.github.io/manager/storybook-static/index.html?path=/story/typography-links--external-link

Reply:
You are right, nice catch thank you